doc/nrf/ug_bt_qualification.rst

```diff
@@ -0,0 +1,101 @@
+To check if your company is already a SIG member, go to `Join Bluetooth SIG`_ page and search the Member Directory list.
+
+Log into `Launch Studio`_, the official qualification tool, that will guide you through one of the following processes depending on the state of your product:
+
+* `Qualification process with no required testing`_ - If your complete product uses only pre-qualified End products or Subsystem products.
+* `Qualification process with required testing`_ - If your complete product uses a new Bluetooth design or pre-qualified Component products.
+
+Updating existing entry
+=======================
+
+Update an existing qualification entry through `Launch Studio`_ in the following cases:
+
+* You significantly changed an existing product's design so that the testing is required.
+* Your product can be added as a new model to one of your existing qualifications.
+* You switched to another SDK version for an existing product.
+
+.. note::
+
+   An existing entry can be edited only by the person that completed the listing.
+   If you want to change the entry owner, contact SIG support.
+
+Switching to another SDK version
+--------------------------------
+
+If you want to switch to another SDK version for an existing product, complete the following steps:
+
+1. Go to the `Nordic Semiconductor Infocenter`_ and search for the SoC or SiP model you use.
+
+#. Navigate to the relevant Compatibility Matrix directory.
+
+#. Open the Bluetooth QDIDs article and, based on the table, confirm the following information:
+
+  * That the version you want to use is compatible with versions of other Subsystems you want to keep for your product.
+  * That relevant Host and SoftDevice Controller Subsystems implemented in the SDK version you want to use are pre-qualified.
+    If yes, check the following:
+
+     * If the version you want to use is fully pre-qualified, and has the same Qualification Design ID (QDID) numbers as the ones currently used for your product, then no further action is required.
+     * If the version you want to use has at least one QDID number that is different from the numbers already used for your product, then a new qualification is required.
+       Log into the `Launch Studio`_ tool and follow the steps for `Qualification process with no required testing`_ option.
+
+Qualification listing
+*********************
+
+The `Bluetooth Qualification Listing`_ search lists all qualified designs.
+Use it in the following cases:
+
+* To check if your qualification process succeeded.
+* To search for existing designs that can be used in your product.
+* To check if you can add your device model to an existing qualification entry.
+
+Matter certification by inheritance
+***********************************
+
+Bluetooth QDIDs may be further used to obtain the Matter certification by inheritence.
+See the :ref:`ug_matter_device_certification_inheritance` section for details.
+
+Support
+*******
+
```

Comment:
Also, it will be nice to add a link to PRD somewhere in this text. 
https://www.bluetooth.com/download/prd_dpd_resolutions/

Reply:
As I have no access there, could you please let me know what the exact article title is and what it is about? As far as I understand, end users should familiarize with PRD and DPD documents before they start the qualification process, am I right? If so, should I mention that article somewhere at the beginning of **Qualification process** section? Please advise on that, thank you!

Reply:
Yes. You are correct. You could say what you mentioned above and give link to these documents.